sdks/java/core/src/test/java/org/apache/beam/sdk/util/HistogramDataTest.java

```diff
@@ -332,4 +332,19 @@ public void testExponentialBuckets_NumBuckets() {
     HistogramData negativeScaleBucket = HistogramData.exponential(-3, 500);
     assertThat(negativeScaleBucket.getBucketType().getNumBuckets(), equalTo(4));
   }
+
+  @Test
+  public void testStatistics() {
```

Comment:
better to split this into 2 tests to test each part of the interface
testStatistics_mean
testStatistics_sumOfSquaredDeviations

Reply:
done